packages/block-library/src/video/editor.scss

```diff
@@ -9,3 +9,8 @@
 .editor-video-poster-control .components-button + .components-button {
 	margin-top: 1em;
 }
+
+.editor-video-poster-control > span {
+	display: block;
+	margin-bottom: 4px;
```

Comment:
Is `BaseControl` truly "unnecessary" when we had to recreate the margin it applies to the label it generates? This has a high risk of falling out of sync, and at the very least is only incidentally aligned to the base control margin (which is assigned by the [value of the `$grid-size-small` variable](https://github.com/WordPress/gutenberg/blob/54e34bf5fc566339cfcce3910b52a19e7a77d27f/packages/components/src/base-control/style.scss#L15), not hard-coded).

I realize in some ways this contradicts what's proposed in https://github.com/WordPress/gutenberg/pull/9802#discussion_r257269125 with having the "just another child" text element.

Maybe the compromise is: Leave it as `BaseControl`, render the label as a `div` / `span`, and effect the margin by some "padding between" all children of the base control field?

```scss
.components-base-control__field > * {
	margin: $grid-size-small 0;

	&:first-child {
		margin-top: 0;
	}

	&:last-child {
		margin-bottom: 0;
	}
}
```

Reply:
Hi @aduth the code was updated following a logic similar to the one you described.
The specific CSS rule is a little bit simpler and just applies the margin to the first child of components-base-control__field, it seems safer e.g: Toggle control works exactly as before while if we apply  "padding between" all children the design get broken.